guard/src/commands/validate/cfn.rs

```diff
@@ -81,19 +81,39 @@ impl<'reporter> Reporter for CfnAware<'reporter> {
         output_type: OutputFormatType,
     ) -> rules::Result<()> {
         let root = data.root().unwrap();
+
         if data.at("/Resources", root).is_ok() {
             let failure_report = simplifed_json_from_root(root_record)?;
             match output_type {
                 OutputFormatType::YAML => serde_yaml::to_writer(write, &failure_report)?,
                 OutputFormatType::JSON => serde_json::to_writer_pretty(write, &failure_report)?,
-                OutputFormatType::SingleLineSummary => single_line(
-                    write,
-                    data_file,
-                    data_file_bytes,
-                    rules_file,
-                    data,
-                    failure_report,
-                )?,
+                OutputFormatType::SingleLineSummary => {
+                    match single_line(
+                        write,
+                        data_file,
+                        data_file_bytes,
+                        rules_file,
+                        data,
+                        failure_report,
+                    ) {
+                        Err(crate::Error::InternalError(_)) => {
+                            self.next.map_or(Ok(()), |next| {
```

Comment:
Does this mean it will skip evaluation for a single line summary and use something like a `generic_summary` instead? Where is the `next` reporter here coming from?

Reply:
Every reporter has a field where it holds the next reporter. If you follow the construction of the reports its like this Cfn -> Tf -> Generic. This means that if cfn calls next, it falls back on terraform, and if it fails the terraform check (which it will since its not a terraform plan) it falls back onto the generic reporter